apis/ec2/src/main/java/org/jclouds/ec2/compute/domain/EC2HardwareBuilder.java

```diff
@@ -290,6 +290,19 @@ private EC2HardwareBuilder hs1() {
       virtualizationTypes(VirtualizationType.HVM, VirtualizationType.PARAVIRTUAL);
       return this;
    }
+   
+   private EC2HardwareBuilder x1() {
+	      virtualizationTypes(VirtualizationType.HVM);
+	      
+	      // TODO X1 is not deprecated, but it requires that you are using a VPC
+	      // until we have a way for hardware instances to be filtered based on network
+	      // we do NOT want X1 selected automatically.
+	      // You get: org.jclouds.aws.AWSResponseException: request POST https://ec2.eu-west-1.amazonaws.com/ HTTP/1.1 failed with code 400, error: AWSError{requestId='2300b99e-ddc0-42ab-b1ed-9d628a161be4', requestToken='null', code='VPCResourceNotSpecified', message='The specified instance type can only be used in a VPC. A subnet ID or network interface ID is required to carry out the request.', context='{Response=, Errors=}'}
+	      // A user can explicitly request a x1 if they are also setting up a VPC.
+	      deprecated();
```

Comment:
[minor] do you think it is still a valid comment? Look quite unusual to not deploy on a VPC these days, is it still important to filter those instance types

Reply:
Thanks @lemmy

@nacx any thoughts on this?

Reply:
May I ask to split the discussion about the deprecation - since it also applies to the existing t2 instance type - into a separate issue? I wait for the x1 support to be included in snapshot builds. Thanks!

Reply:
I think it makes sense. I'll merge it asap

Reply:
Thanks

Reply:
Yep. Instead of abusing the `deprecated` thing we should add a method that says `requiresVPC()` or similar to properly mark those hardware profiles, and remove that ugly and confusing comment, but that's for another PR.